### PR TITLE
fix(type): Add the missing types to the ”accounts“

### PR DESCRIPTION
**Pull request by electroluxcode**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

According to the documentation at https://docs.web3js.org/libdocs/Accounts, add the missing types to the account

### CODE REVIEW

File: packages/web3/src/types.ts [hunk 1]
```diff
@@ -75,6 +75,9 @@ export interface Web3EthInterface extends Eth {
 	accounts: {
 		create: () => Web3Account;
 		privateKeyToAccount: (privateKey: Uint8Array | string) => Web3Account;
+		privateKeyToAddress: (privateKey: Uint8Array | string) => string;
+		privateKeyToPublicKey: (privateKey: Uint8Array | string,isCompressed:boolean) => string;
+		parseAndValidatePrivateKey:(data: Uint8Array | string,ignoreLength?:boolean) => Uint8Array;
```
Comment (Muhammad-Altabba) [inline packages/web3/src/types.ts]: Using the exact types as in: https://github.com/web3/web3.js/blob/4.x/packages/web3-eth-accounts/src/account.ts#L440 for readability:

```suggestion
		privateKeyToAddress: (privateKey: Bytes) => string;
		privateKeyToPublicKey: (privateKey: Bytes, isCompressed: boolean) => string;
		parseAndValidatePrivateKey:(data: Bytes, ignoreLength?: boolean) => Uint8Array;
```

**Review by Muhammad-Altabba — CHANGES_REQUESTED** [review]
Thanks @electroluxcode for your catch and the MR.
Could you please respond to the suggestion below. And update the `packages/web3/CHANGELOG.md` file?

It will be also nice and helpful to write small Unit tests inside the web3 package for those added functions.

**Review by Muhammad-Altabba — APPROVED** [review]

**Review by avkos — APPROVED** [review]

**Review by avkos — CHANGES_REQUESTED** [review]
LGTM. 
Thanks for your contribution.
Please fix lint test

**Comment by jdevcs** [comment]
> LGTM. Thanks for your contribution. Please fix lint test

@electroluxcode could you fix conflicting file and above comments, Thanks

**Comment by jdevcs** [comment]
closing this PR as it will be merged via https://github.com/web3/web3.js/pull/7011 Thanks @electroluxcode

**Pull request closed without merging**